src/other_libs/dlgmodule/macos/macos.cpp

```diff
@@ -0,0 +1,299 @@
+/*
+
+ MIT License
+
+ Copyright © 2020 Samuel Venable
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+*/
+
+#include "../../dlgmodule.h"
+#include "buttons.h"
+#include "config.h"
+#include <sys/stat.h>
+#include <climits>
+#include <cstdlib>
+#include <cstring>
+#include <cstdio>
+#include <fstream>
+#include <string>
+
+using std::string;
+
+const char *cocoa_widget_get_button_name(int type) {
+  return dialog_module::widget_get_button_name(type);
```

Comment:
i forgot to remove the "dialog_module::" from this line.

Reply:
```suggestion
  return widget_get_button_name(type);
```
Like this?

Reply:
yes